linera-execution/src/runtime.rs

```diff
@@ -589,7 +596,15 @@ where
             .await
             .remove(&self.application_id())
         {
-            Some(mut view) => Ok(view.write_batch(batch).await?),
+            Some(mut view) => {
+                let stored_size = view.total_size().sum() as i32;
+                view.write_batch(batch).await?;
+                let new_stored_size = view.total_size().sum() as i32;
```

Comment:
We could use safe conversions here, too, instead of `as i32`.

Reply:
ok.